Client/ServiceClient.cs

```diff
@@ -71,11 +71,11 @@ private static double ToUnixTimeStamp(DateTime date)
 
         private Action<HttpWebRequest> requestFilter;
 
-        public bool Login(string userName, string password, string validationCode)
+        public bool Login(string userName, string password, string validationCode, string captcha)
```

Comment:
The captcha argument needs a default value to become optional. Same for the other Login() overload.